src/Components/DetailPage/DetailSupport/DetailSupport.js

```diff
@@ -0,0 +1,181 @@
+import React from "react";
+import styled from "styled-components";
+
+const DetailSupport = () => {
+  return (
+    <SupportBox>
+      <SupportHeader>
+        <div></div>
+        <h2>지원하기 </h2>
+        <button>뒤로</button>
+      </SupportHeader>
+      <SupportBody>
+        <SupportSubHeader>지원 정보</SupportSubHeader>
+        <CommonInput>
+          <h4>이름</h4>
+          <input></input>
+        </CommonInput>
+        <CommonInput>
+          <h4>이메일</h4>
+          <input></input>
+        </CommonInput>
+        <CommonInput>
+          <h4>연락처</h4>
+          <input></input>
+        </CommonInput>
+        <CommonInput>
+          <h4>추천인</h4>
+          <input></input>
+        </CommonInput>
+        <SupportSubHeader>첨부 파일</SupportSubHeader>
+        <SupportBtn>
+          <LabelBox>
+            <label for="ex_file">업로드</label>
+            <input type="file" id="ex_file"></input>
```

Comment:
```suggestion
            <label>
              <span>업로드</span>
              <input type="file" />
            </label>
```
- 가능하면 텍스트는 span, p 태그등으로 감싸주세요
- label로 input 감싸면 for, id 불필요합니다